src/main/java/io/cdap/plugin/google/shopping/content/source/ShoppingContentConfig.java

```diff
@@ -0,0 +1,47 @@
+package io.cdap.plugin.google.shopping.content.source;
+
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Macro;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.plugin.PluginConfig;
+import io.cdap.cdap.etl.api.validation.InvalidConfigPropertyException;
+import java.math.BigInteger;
+
+/**
+ * Configurations for making Shopping Content API calls.
+ */
+public class ShoppingContentConfig extends PluginConfig {
+
+  @Name("referenceName")
+  @Description("This will be used to uniquely identify this source/sink for lineage, annotating"
+      + " metadata, etc.")
+  public String referenceName;
+
+  @Name("merchantID")
+  @Description("The unique identifier of the merchant.")
+  @Macro
+  private String merchantId;
+
+  @Name("serviceAccountPath")
+  @Description("The path to locate service-account.json file which is used to authenticate with "
+      + "Shopping Content API.")
+  @Macro
+  private String serviceAccountPath;
+
+  public String getMerchantId() {
```

Comment:
it's better for this getter to return a BigInteger, so that the caller always knows its valid.